cmd/introspect/baml_parser_golden_test.go

```
@@ -0,0 +1,413 @@
+		return "map[string]bedrockClientOptionsSnapshot{}"
+	}
+	keys := make([]string, 0, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+	var b bytes.Buffer
+	b.WriteString("map[string]bedrockClientOptionsSnapshot{\n")
+	for _, k := range keys {
+		opts := m[k]
+		b.WriteString(fmt.Sprintf("%q: {\n", k))
```

golangci-lint (2.12.2)

[error] QF1012: Use fmt.Fprintf(...) instead of WriteString(fmt.Sprintf(...))

(staticcheck)